packages/open-next/src/types/open-next.ts

```diff
@@ -313,6 +318,16 @@ export interface FunctionOptions extends DefaultFunctionOptions {
    * @deprecated This is not supported in 14.2+
    */
   experimentalBundledNextServer?: boolean;
+
+  /**
+   * The route preloading behavior. Only supported in Next 15+.
+   * - "none" - No preloading of the route at all
```

Comment:
Move the comments on the type l121?

IIUC "none" is the default behavior of next (/OpenNext):
- maybe this should be explicitly documented
- is "none" accurate if it happens in the background ?